Controllers/OrdersController.cs

```diff
@@ -37,14 +39,18 @@ public async Task<ActionResult<IEnumerable<GetDataDTO<OrdersDTO>>>> GetOrders()
 
 
         [Authorize(Roles = "Warehouse")]
-        [HttpGet("{id}")]
-        public async Task<ActionResult<GetDataDTO<OrdersDTO>>> GetOrdersByWarehouse(int id)
+        [HttpGet("byID")]
+        public async Task<ActionResult<GetDataDTO<OrdersDTO>>> GetOrdersByWarehouse()
         {
             if (!IsValidApiRequest())
             {
                 return ApiBadRequest("Invalid Headers!");
             }
 
+            string email = User.FindFirst(ClaimTypes.Email).Value;
+            var user = await UserManager.FindByEmailAsync(email);
```

Comment:
Maybe we could use `await GetCurrentUser();` here instead to retrieve user?

Reply:
Done.